### ISSUE TITLE
Lesson Contribution - Moving paragraphs to clarify R vs RStudio

### ISSUE THREAD

**Issue by klbarnes20** [datacarpentry/r-intro-geospatial#113]
I'm a member of The Carpentries Core Team and I'm submitting this issue on behalf of another member of the community. In most cases, I won't be able to follow up or provide more details other than what I'm providing below.
---------------------------
In the Introduction to R and RStudio episode, I suggest moving the first two paragraphs of the "Intro to R" section to the "Before Starting the Workshop" section or make a new section called "R versus RStudio." Then explain that R is the language you will use and RStudio is the editor that you will work with the R language in. When I first started R this year, I had no idea there was a difference, so it might be worth distinguishing them for novice learners at the very beginning. Then explain that R is often used to refer to both the programming language and RStudio, even within this lesson.

**Comment by albhasan** [comment]
I disagree with this suggestion becase the section "Before Starting the Workshop" happens before the lesson and while "Introduction to Rstudio" starts by explainng what R and Rstudio are. 

In other words, "Before Starting the Workshop" is a requirements section while "Introduction to Rstudio" is a proper part of the lesson and it's better to explain concepts during the lesson.

For that reason, I suggest to close this issue.
